satpy/etc/readers/mersi_ll_l1b.yaml

```diff
@@ -0,0 +1,251 @@
+reader:
+  description: FY-3E Medium Resolution Spectral Imager - Low Light (MERSI-LL) L1B Reader
+  name: mersi_ll_l1b
+  sensors: [mersi-ll]
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+
+file_types:
+  mersi_ll_l1b_1000:
+    file_reader: !!python/name:satpy.readers.mersi_l1b.MERSIL1B
+    rows_per_scan: 10
+    file_patterns:
+      # From National Meteorological Satellite Center
+      - '{platform_shortname}_MERSI_GRAN_L1_{start_time:%Y%m%d_%H%M}_1000M_V0.{ext:3s}'
+      # Generic
+      - '{filename_mda}_MERSI_1000M_L1B.{ext:3s}'
+
+  mersi_ll_l1b_250:
+    file_reader: !!python/name:satpy.readers.mersi_l1b.MERSIL1B
+    rows_per_scan: 40
+    file_patterns:
+      # From National Meteorological Satellite Center
+      - '{platform_shortname}_MERSI_GRAN_L1_{start_time:%Y%m%d_%H%M}_0250M_V0.{ext:3s}'
+      # Generic
+      - '{filename_mda}_MERSI_0250M_L1B.{ext:3s}'
+
+  mersi_ll_l1b_1000_geo:
+    file_reader: !!python/name:satpy.readers.mersi_l1b.MERSIL1B
+    rows_per_scan: 10
+    file_patterns:
+      # From National Meteorological Satellite Center
+      - '{platform_shortname}_MERSI_GRAN_L1_{start_time:%Y%m%d_%H%M}_GEO1K_V0.{ext:3s}'
+      # Generic
+      - '{filename_mda}_MERSI_GEO1K_L1B.{ext:3s}'
+
+  mersi_ll_l1b_250_geo:
+    file_reader: !!python/name:satpy.readers.mersi_l1b.MERSIL1B
+    rows_per_scan: 40
+    file_patterns:
+      # From National Meteorological Satellite Center
+      - '{platform_shortname}_MERSI_GRAN_L1_{start_time:%Y%m%d_%H%M}_GEOQK_V0.{ext:3s}'
+      # Generic
+      - '{filename_mda}_MERSI_GEOQK_L1B.{ext:3s}'
+
+
+# NOTE: OSCAR website currently has bands in wavelength order
+#       https://www.wmo-sat.info/oscar/instruments/view/279
+#       The order below is by the wavelength in the input files
+#       The slides at the below link have band 5 and 19 swapped:
+#       http://www.wmo.int/pages/prog/sat/meetings/documents/IPET-SUP-4_Doc_05-04_FY-3D-ppt.pdf
```

Comment:
How much do we know about what might change with the band naming/ordering? Should we wait a bit longer to merge this? Should we use different band names than just the numbers?

Reply:
This was from the original MERSI-2 reader, it's not something I added.

I should delete this comment from the YAML file, as it only applies to MERSI-2.

Reply:
Yeah I guess it would make sense to get rid of it for the LL reader. Thanks.